src/HttpClient/JanusClient.ts

```diff
@@ -3,6 +3,11 @@ import { IOContext } from '../service/typings'
 import { ExternalClient } from './ExternalClient'
 import { InstanceOptions } from './typings'
 
+enum Environment {
```

Comment:
why not using string type literal ?
```suggestion
type Environment = 'beta' | 'stable' 
```

Reply:
I do prefer enum. For me is more readable and easier to write.